<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WaniKani</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/icon</normaloff>:/icon</iconset>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QGroupBox" name="apiKeyGroupBox">
     <property name="title">
      <string>API Key</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="apiKeyValue"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="updateGroupBox">
     <property name="title">
      <string>Update</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSpinBox" name="intervalSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string> min</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="forceUpdateButton">
        <property name="text">
         <string>Force Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fontGroupBox">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFontComboBox" name="fontComboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="writingSystem">
         <enum>QFontDatabase::Japanese</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="boldFontCheckBox">
          <property name="text">
           <string>Bold</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="italicsFontCheckBox">
          <property name="text">
           <string>Italics</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
